kerlescan/view_helpers.py

```diff
@@ -171,11 +171,14 @@ def ensure_has_permission(**kwargs):
 
     try:
         perms = get_perms(
-            kwargs["application"],
+            ",".join(
+                [kwargs["application"], "inventory"]
+            ),  # we need to query permissions for the inventory app
             auth_key,
             logger,
             kwargs["request_metric"],
             kwargs["exception_metric"],
+            kwargs.get("flask_g", None),
```

Comment:
Do we need to pass the whole Flask's global to this method? Can we just pass what's necessary?

Reply:
Good catch, thank you. Solved.